src/utils/jwt.js

```diff
@@ -33,7 +33,9 @@ let JwtTokenInternalDef;
  * Provides helper methods to decode and verify JWT tokens.
  */
 export class JwtHelper {
-
+  /**
```

Comment:
Remove this one. Really not a big fun of generic comments.

Reply:
Done